.github/script/call-canary-deploy-demo-prod

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+const { spawnOrFail } = require('../../script/cli-utils');
+
+const myArgs = process.argv.splice(2);
+var demo_current_version = myArgs[0];
+
+var npm_latest_version = spawnOrFail('npm',['view', 'amazon-chime-sdk-js@latest', 'version']);
+var npm_beta_version = spawnOrFail('npm', ['view', 'amazon-chime-sdk-js@beta', 'version']);
+
+console.log('NPM latest version: ', npm_latest_version);
+console.log('NPM beta version: ', npm_beta_version);
+console.log('Demo current version: ', demo_current_version);
+
+// Removing line breaks from the strings before comparison
+npm_latest_version = npm_latest_version.replace(/(\r\n|\n|\r)/gm,'');
+npm_beta_version = npm_beta_version.replace(/(\r\n|\n|\r)/gm,'');
+demo_current_version = demo_current_version.replace(/(\r\n|\n|\r)/gm,'');
+
+if (npm_beta_version == npm_latest_version || npm_beta_version == '') {
```

Comment:
General `if..else` pattern is in below format in other scripts, so suggest using the same for consistency:
```
if (condition) {
} else if (condition) {
} else {
}
```

Reply:
Done